src/src/main/java/neonracer/gui/widget/grid/Grid.java

```diff
@@ -0,0 +1,38 @@
+package neonracer.gui.widget.grid;
+
+import neonracer.gui.util.PropertyList;
+import neonracer.gui.widget.base.Container;
+import neonracer.gui.widget.base.Widget;
+
+public class Grid extends Container {
```

Comment:
Wenn es mal Listen mit mehreren Einträgen gibt, wäre es wichtig, in beide Richtungen scrollen zu können.

Reply:
Wieso muss ich das Grid scrollen können, wenn ich Listen mit vielen Einträgen brauche? Scrolling ist bisher nicht implementiert.

Reply:
Dann wird dieses Feature verschoben und bei Bedarf später hinzugefügt.